assets/base.css

```diff
@@ -3225,6 +3224,18 @@ details-disclosure > details {
     transform: rotate(0.5deg); /* Less intense rotation for collage items. */
   }
 
+  .animate--hover-3d-lift .product-grid .grid__item:hover,
+  .animate--hover-3d-lift .collection-list .grid__item:hover,
+  .animate--hover-3d-lift .collage__item:hover,
+  .animate--hover-3d-lift .blog-articles .article:hover,
+  .animate--hover-3d-lift .complementary-slide li:hover {
+    z-index: 2; /* Make sure the hovered card is the topmost card. */
```

Comment:
Giving the hovered item a bigger z-index does tend to create a flickering effect in some cases since z-index is either 0 or 2. It would be pretty tricky to animate the shadow, though. I can imagine fancy and complicated ways of doing it using two shadows: a top shadow and a bottom shadow that fade in and out depending on hover.

https://user-images.githubusercontent.com/14841718/235761801-b8018379-cb1b-4868-a8cb-d5ddad301769.mov